streaming/src/test/scala/org/apache/spark/streaming/UISeleniumSuite.scala

```diff
@@ -77,7 +77,14 @@ class UISeleniumSuite
     inputStream.foreachRDD { rdd =>
       rdd.foreach(_ => {})
       try {
-        rdd.foreach(_ => throw new RuntimeException("Oops"))
+        rdd.foreach { _ =>
+          import org.apache.spark.TaskContext
```

Comment:
This should be together with other imports. Will fix during merge.

Reply:
Thanks!